src/App.js

```diff
@@ -35,7 +35,7 @@ const App = () => {
       .getUser()
       .then(() =>
         getRbacRoleApi()
-        .listRoles(defaultSettings.limit, 0, 'Approval ', 'partial', 'principal')
+        .listRoles(defaultSettings.limit, 0, 'Approval ', '', 'partial', 'principal')
```

Comment:
I assume there was a change in the client arguments so you have to include the empty string?

Reply:
Yes, a new parameter was added to this endpoint, so this new param is needed